fastlane/lib/fastlane/actions/app_store_build_info.rb

```diff
@@ -180,7 +180,7 @@ def self.example_code
       end
 
       def self.authors
-        ["rishabhtayal"]
+        ["hjanuschka"]
```

Comment:
You can keep your name here too since you changed stuff! But just wanted to keep `hjanuschka` in there too 😉